invenio_app_ils/webpack.py

```diff
@@ -36,6 +36,7 @@
         "redux-thunk": "^2.3.0",
         "semantic-ui-calendar-react": "^0.15.3",
         "semantic-ui-css": "^2.4.1",
+        "semantic-ui-less": "^2.4.1",
```

Comment:
did you work on adding less in pywebpack?
today we have the meeting to discuss about the SPA, it might not be needed :)